src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -205,13 +199,13 @@ def body(self):
         self.cmd('network express-route circuit list --resource-group {}'.format(self.resource_group),
             checks=NoneCheck())
 
-class NetworkExpressRouteCircuitScenarioTest(VCRTestBase):
+class NetworkExpressRouteCircuitScenarioTest(ResourceGroupVCRTestBase):
```

Comment:
Since we use TODOs for tracking such changes, I added a TODO.
Our tests should move from `VCRTestBase` to `ResourceGroupVCRTestBase` and I think a TODO is a better way of tracking that changing it back to `VCRTestBase`.